lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -167,6 +168,7 @@ impl TeleportRdpdrBackend {
 }
 
 /// A generic error type for the TeleportRdpdrBackend that can contain any arbitrary error message.
+#[allow(dead_code)]
```

Comment:
Why do we need this. Is this truly unused code?

Reply:
It's false positive, `clippy` complaints about unused `String` but it's used in derived `Debug`

Reply:
Could we explain with a comment?

Same for all others.